### ISSUE TITLE
Add support for sixth level of headings

### ISSUE THREAD

**Issue by racke** [melmothx/text-amuse#53]
HTML headings have six levels, so I would be useful to have sixth level supported by Text::Amuse.

**Comment by melmothx** [comment]
@racke We're alread go to the `h6`, with `h1` reserved for the text title.

**Comment by racke** [comment]
Than it seems well hidden, there is no test for `h6` and `******` didn't seem to work in Amusewiki.

**Comment by melmothx** [comment]
well, `*****` maps to h6. It's basically number of stars + 1. 6 stars would be h7, which doesn't exist

**Comment by racke** [comment]
OK ... in this case the edit box in Amusewiki is misleading because it offers h1...h5.

**Comment by melmothx** [comment]
Yes, I agree that it's misleading. The logic was that it refers to the muse level, not to the the html ones, but I acknowledge the problem. Probably that editor should be replaced with something fresher as well...

**Comment by melmothx** [comment]
Closing here, moved issue to amusewiki, where it belongs.
